charms/jupyter-controller/terraform/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "jupyter-controller"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
```

Comment:
For dex-auth, I used the actual channel of the target version, so in the notebooks case, 1.8/stable. Should we change that here?